src/table/table.scss

```diff
@@ -16,23 +16,31 @@
   @include iui-table-cell-icon;
 }
 
+@mixin iui-table-header-wrapper {
+  overflow: hidden;
+  display: flex;
+  flex-shrink: 0;
+}
+
 @mixin iui-table-header {
+  display: flex;
   user-select: none;
-  overflow: hidden;
+  min-width: 100%;
```

Comment:
Can you explain this width stuff?

Reply:
Without it:
![image](https://user-images.githubusercontent.com/36186912/167134798-59436ff9-b263-4f19-885b-898568cb05d3.png)

Reply:
ah ok I see `iui-row` also has it. But then how was header working before?

Reply:
I think it is now needed because of the header wrapper.